reference/toolstacks/php/drupal/redis-drupal-8.md

```diff
@@ -0,0 +1,154 @@
+# Using Redis with Drupal 8.x
+
+> **note**
+> Redis support for Drupal 8 is still in alpha. The instructions below assume
+> you are OK with running an alpha-version of a Drupal module. It should be
+> stable and has been deployed in production, but more cautious users may prefer
+> to wait for the module to have a stable release. The instructions below are
+> unlikely to change at that time, but it is possible. See https://www.drupal.org/project/redis 
+> for more information.
+
+The Drupal 8 Redis module currently only supports the [PhpRedis](https://github.com/nicolasff/phpredis)
+option, which relies on a PHP extension. Fortunately, that extension is trivial to enable on
+Platform.sh
+
+## Requirements
+
+### Add a Redis service
+
+First you need to create a Redis service.  In your `.platform/services.yaml` file,
+add or uncomment the following:
+
+```yaml
+rediscache:
+    type: redis:3.0
+```
+
+That will create a service named `rediscache`, of type `redis`, specifically version `3.0`.
+
+### Expose the Redis service to your application
+
+In your `.platform.app.yaml` file, we now need to open a connection to the new 
+Redis service.  Under the `relationships` section, add the following:
+
+```yaml
+relationships:
+    redis: "rediscache:redis"
+```
+
+The key (left side) is the name that will be exposed to the application in the PLATFORM_RELATIONSHIPS
+variable.  The right hand side is the name of the service we specified above (`rediscache`) and
+the type of endpoint (`redis`).  If you named the service something different above, change `rediscache`
+to that.
+
+### Add the Redis PHP extension
+
+If you're using PhpRedis you will need to enable the PHP Redis extension.  In your `.platform.app.yaml` file,
+add the following right after the `type` block:
+
+```yaml
+# Additional extensions
+runtime:
+    extensions:
+        - redis
+```
+
+### Add the Drupal module
+
+You will need to add the [Redis](https://www.drupal.org/project/redis)
+module to your project.  If you are using Composer to manage your Drupal 8 site
+(which we recommend), simply run:
+
+```bash
+composer require drupal/redis
+```
+
+Then commit the resulting changes to your `composer.json` and `composer.lock` files.
+
+## Configuration
+
+To make use of the Redis cache you will need to set some Drupal variables. The
+configuration is a bit more complex than can be easily represented in
+Platform.sh's environment variables configuration, so using `settings.php` directly
+is the recommended approach.
+
+Place the following at the end of `settings.php`, after the include directive
+for `settings.local.php`.  Note the inline comments, as you may wish to customize
```

Comment:
It looks like that's what we recommend in the Drupal 7 instructions, so I agree it makes sense to be consistent.